README.md

```diff
@@ -49,3 +49,79 @@ npm run lint
 
 
 [https://nodejs.org/en/download/]: https://nodejs.org/en/download/
+
+
+Token migration further explained
+
+Alongside the token migration app (source here and also available on https://token-migration.aepps.com/) 
+we have built a host of additional tools which give the user a smooth migration journey as well as 
+integrating with 3rd party wallets (as mentioned above).
+
+Note: We refer often to the term "Token Burning" which simply means the transfer of ERC20 tokens from etherium
+blockchain to the aeternit main net.
+
+A brief explanation of the token transfer process.
+
+To transfer tokens it is necessary to have enough ETH to pay for gas and transaction fees. Tokens can be transferred
+in unlimited multiple batches. When a user transfers his/her tokens, first they specify the number, 2nd they connect
+their wallet to sign the transaction. In the 3rd step the contract burner contract (see below) a solidity smart contract is invoked which receives and prepares the tokens for their availability on the aeternity main net.
+
+Tools (We built and use)
+
+1. Token migration landing page: This is the home page that token owners got to, to begin their token migration.
```

Comment:
> Token migration landing page: This is the home page that token owners got to, to begin their token migration.

Maybe better like: `Token migration landing page: This is the home page that token owners __go to begin__ their token migration.`